miner/bid_simulator.go

```diff
@@ -535,19 +544,20 @@ func (b *bidSimulator) simBid(interruptCh chan int32, bidRuntime *BidRuntime) {
 		if success {
 			bidRuntime.duration = time.Since(simStart)
 			bidSimTimer.UpdateSince(simStart)
+			metrics.GetOrRegisterCounter(fmt.Sprintf("bid/sim/count/%d", bidRuntime.bid.BlockNumber), nil).Inc(1)
```

Comment:
it could have impact on metric efficiency, as each block would have registry a metric, there would be ~28,800(24*60*60/3) metrics registered per day, it is too much, it is not recommended.

Reply:
ok，removed